src/components/pie-player/pie-player.tsx

```diff
@@ -407,6 +426,11 @@ export class Player {
         }
       }
     }
+
+    if (this.showBottomBorder && this.env.mode === 'evaluate') {
```

Comment:
I imagined this has to be in the "else" statement above (does it?). Or do we need elementsLoaded to be true? And if that's the case, can we put this in the "if (this.elementsLoaded) " statement?